web/modules/custom/dpl_react_apps/dpl_react_apps.routing.yml

```diff
@@ -18,3 +18,12 @@ dpl_react_apps.work:
     _controller: '\Drupal\dpl_react_apps\Controller\DplReactAppsController::work'
   requirements:
     _permission: 'access content'
+
+dpl_react_apps.reader:
+  path: '/reader'
+  defaults:
+    _controller: '\Drupal\dpl_react_apps\Controller\DplReactAppsController::reader'
+  requirements:
+    _permission: 'access content'
+  options:
+    no_cache: true
```

Comment:
Please reconsider if there is a reason to add `no_cache`.

We do not do so for other apps which I consider similar so I would not except it to be necessary here either.